src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -692,7 +692,7 @@ private JabRefPreferences() {
         // Download files by default
         defaults.put(DOWNLOAD_LINKED_FILES, true);
         // Create Fulltext-Index by default
-        defaults.put(FULLTEXT_INDEX_LINKED_FILES, true);
+        defaults.put(FULLTEXT_INDEX_LINKED_FILES, false);
```

Comment:
Actually I would leave this as true and only the search button deactivated for the moment

Reply:
It's what we literally discussed yesterday and decided againstindexing fulltext by default.

Reply:
I personally changed my mind after the meeting.

If indexing is disabled, users will try to use file search, but it will not work. Users will not know that they first have to enable indexing for file search to work. They will report that file search is broken. Disable indexing only, once users are somehow notified that file search can only be used, if indexing is enabled.

Reply:
yes we discussed also that this is not to be implemented as the lucene search pr of @btut changes so many things with the search that there should not be made any major changes in behaviour, GUI or logic, since all this work would be thrown away when or have to be undone before merging the lucene PR.
Decisions were made for a reason yesterday. Don't start rediscussing what we already decided yesterday.

Reply:
Yesterdays discussion notes: https://github.com/JabRef/jabref/issues/9491#issuecomment-1369188282